apps/browser/css/navbar.css

```diff
@@ -78,6 +88,14 @@
 .reload-button {  background-position: -73px   0px; }
 .stop-button {    background-position: -90px   0px; }
 .menu-button {    background-position: -468px  0px; }
+  
+   /* styles for the urlbar buttons */
+
+.urlbar > .reload-button { background-position: 2px -1px; }
+.urlbar > .reload-button:active { background-position: 2px -15px; }
+.urlbar > .stop-button { background-position: -12px 0px; }
+.urlbar > .stop-button:active { background-position: -12px -14px; }
+ 
```

Comment:
These are weird positions. Use the proper coordinates of the images (0px/14px 0px/14px).